### PR TITLE
Feature/fwf 3736 preview page

### PR DESCRIPTION
**Pull request by Ajay-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-3736
Issue Type: BUG/ FEATURE

# Changes
added form and workflow  preview page for users with view design permisson only

# Screenshots (if applicable)
image
image

# Checklist
- [ ] Updated changelog
- [ ] Added meaningful title for pull request

### CODE REVIEW

**Review by arun-s-aot — APPROVED** [review]

File: forms-flow-web/src/components/Form/EditForm/FormEdit.js [hunk 1]
```diff
@@ -946,65 +948,77 @@ const EditComponent = () => {
...
-                      onChange={formChange}
-                      options={{
-                        language: lang,
-                        i18n: RESOURCE_BUNDLES_DATA,
-                      }}
-                    />
+                    {!createDesigns ? (
+                      <div className="px-4 pt-4 form-preview">
+                        <Form
+                          form={form}
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/Form/EditForm/FormEdit.js:1008]: need to pass options too

Comment (shuhaib-aot) [reply forms-flow-web/src/components/Form/EditForm/FormEdit.js:1008]: in options need to pass lang

**Review by shuhaib-aot — CHANGES_REQUESTED** [review]
there are couple of changes.
1. if there is no designer permission for flow render diagram preview component 
2. if there is no designer permission for form render interactive form component

File: forms-flow-web/src/components/Form/EditForm/FormEdit.js (hunk 1 above)
Comment (Ajay-aot) [reply forms-flow-web/src/components/Form/EditForm/FormEdit.js:1008]: added

**Review by Bonymol-aot — APPROVED** [review]

**Comment by arun-s-aot** [comment]
Please chjeck sonarcloud issue @Ajay-aot

**Comment by arun-s-aot** [comment]
Sonar cloud issues to be taken care in @Josephalexantony-aot  commit. Merging the PR for now @Ajay-aot

**Review by arun-s-aot — APPROVED** [review]

**Pull request merged**